bundle/edu.gemini.itc.shared/src/main/scala/edu/gemini/itc/shared/Parameters.scala

```diff
@@ -38,7 +38,14 @@ sealed trait Spectroscopy
 
 sealed trait CalculationMethod {
   def exposureTime: Double
+  def coadds: Option[Int]
+  /*
+     It seems that using .getOrElse is not straightforward to use from Java,
+     so we can use this "coaddsOrElse" to simplify it.
+   */
+  def coaddsOrElse(d: Int): Int = coadds.getOrElse(d)
```

Comment:
What happens, out of curiosity, when you try to use `getOrElse` from Scala?
I would think maybe you need to turn Scala `Int` to Java `java.lang.Integer`?
This should be fine, though.

Reply:
I think the problem was that getOrElse required a scala.Function0<…> argument, and I was not sure how to implement that.